modules/mtc-vm-creating-mig-plan.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+// * migration_toolkit_for_containers/mtc-migrating-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="mtc-vm-creating-mig-plan_{context}"]
+= Creating the migration plan
+
+.Procedure
+
+* To migrate the `mig-vm` namespace, ensure that the `namespaces` field of the migration plan includes `mig-vm`.
++
+Modify the contents of the migration plan by adding `mig-vm` to the namespaces.
++
+.Example YAML
++
+[source,yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigPlan
+metadata:
+  name: live-migrate-plan
+  namespace: openshift-migration
+spec:
+  namespaces:
+  - mig-vm # <1>
+...
+----
++
+<1> add `mig-vm` to the namespaces.
+
+* In order to attempt a live storage migration, the `liveMigrate` field in the migration plan specification must be set to true, and KubeVirt must be configred and be enabled to perform live storage migration.
++
+[source,yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigPlan
+metadata:
+  name: live-migrate-plan
+  namespace: openshift-migration
+spec:
+  liveMigrate: true
+  namespaces:
+...
+----
+
+* Live migration only happens during the cutover of a migration plan. Staging the migration plan skips any running virtual machines and does not sync the data. Any stopped virtual machine disks will be synced.
```

Comment:
I don't think that lines 46-58 belong in this procedure module. Maybe they belong in a separate module that highlights what goes on "behind the scenes", like what I did here. https://docs.openshift.com/container-platform/4.16/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.html#how-the-live-migration-process-works_migrate-from-openshift-sdn